### PR TITLE
TheMediaGrid: added app type support

### PR DESCRIPTION
**Pull request by TheMediaGrid**

### CODE REVIEW

File: static/bidder-info/grid.yaml [hunk 1]
```diff
@@ -5,3 +5,7 @@ capabilities:
     mediaTypes:
       - banner
       - video
+  app:
+    mediaTypes:
+      - banner
+      - video
```
Comment (guscarreon) [inline static/bidder-info/grid.yaml:11]: Nitpick: can `app` come first in the list please?
```
capabilities:
  app:
    mediaTypes:
      - banner
      - video
  site:
    mediaTypes:
      - banner
      - video
```

Comment (TheMediaGrid) [reply static/bidder-info/grid.yaml]: Done

**Comment by TheMediaGrid** [comment]
Just updated. Any chance this could be merged and released today?

**Comment by guscarreon** [comment]
> Just updated. Any chance this could be merged and released today?

@TheMediaGrid we have a freeze until Monday. I'm sorry but our next release will be Monday morning.

**Comment by TheMediaGrid** [comment]
Any chance this will be released this week? I thought the coming release should've been on Monday morning.

**Comment by SyntaxNode** [comment]
This PR will be included in our release tomorrow. Last week's release was cut earlier than usual due to the US July 4th holiday.

**Pull request merged**